ClavaWeaver/src/pt/up/fe/specs/clava/weaver/CxxWeaver.java

```diff
@@ -57,22 +53,19 @@
 
 /**
  * Weaver Implementation for CxxWeaver<br>
- * Since the generated abstract classes are always overwritten, their implementation should be done by extending those
+ * Since the generated abstract classes are always overwritten, their
+ * implementation should be done by extending those
```

Comment:
Just a reminder that we will be generating the concrete classes that extend the abstract classes, if they do not exist. Check if an issue has been created for this in lara-framework, this would imply updating this comment that is generated by WeaverGenerator.

Reply:
When that change to the WeaverGenerator materializes, we'll do that.